test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -190,15 +216,22 @@ private static string GetBuildArgs(params string[] args)
             return buildArgs;
         }
 
-        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant)
+        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant, string architecture)
         {
             string variantName = Enum.GetName(typeof(DotNetImageType), imageType).ToLowerInvariant().Replace('_', '-');
+            bool isArmRuntime = String.Equals("arm", architecture, StringComparison.OrdinalIgnoreCase) && !String.Equals("sdk", variantName, StringComparison.OrdinalIgnoreCase);
```

Comment:
In order to keep the tests data driven, I feel like it might be better to add an SdkOsVariant to the ImageDescriptor and flow that in as appropriate to this method.  This will give us the needed flexibility as we move forward.